src/cli.js

```diff
@@ -5,18 +5,31 @@ const program = require('commander')
 const pkg = require('../package.json')
 const stackdriver = require('././index')
 
+function collect (value, previous) {
+  return previous.concat([value])
+}
+
 // main cli logic
 function main () {
   program
     .version(pkg.version)
     .option('-c, --credentials <credentials>', 'The file path of the JSON file that contains your service account key')
     .option('-p, --project <project>', 'Your Google Cloud Platform project ID')
-    .action(({ credentials, project }) => {
+    .option('-k, --key <key:customKey>', 'Customize additional data to include in log metadata', collect, [])
```

Comment:
Could you add a test for the new option so we'll have 100% test coverage again?

Reply:
Shouldn't it be covered by this test? https://github.com/ovhemert/pino-stackdriver/pull/54/files#diff-da8fe3f2391da380ddef00afcc0f93f0R54

Reply:
Ah I see - I've added an additional error case in 776b6e2d12e434cd2159e2dd0d063463bcd9e62f

<img width="848" alt="image" src="https://user-images.githubusercontent.com/23356519/79596570-a5e5ba80-8095-11ea-9554-ba57a88c70d8.png">